repository/a.json

```diff
@@ -2387,6 +2387,17 @@
 				}
 			]
 		},
+		{
+			"name": "AutoSplit",
+			"details": "https://github.com/egnartsms/sublime-autosplit",
+			"author": "Serhii Mozghovyi",
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
If it's a sublime text 3 package, please indicate a required sublime_text version number >3000.

Reply:
Thank you for valuable suggestions! Done.